client/js/analysis.js

```diff
@@ -1,15 +1,123 @@
-// Listen for on submit click
-let formSubmit = document.getElementById("formSubmit");
-formSubmit.addEventListener("click", fetchAndShowResponse, false);
-
-/**
- * Grabs response from server
- */ 
-async function fetchAndShowResponse() {
-  const response = await fetch('/api/musix');
-  const value = await response.json();
-  const el = document.getElementById(elId); 
-  el.innerText = value;
-  return value;
+window.onload = function() {
+  for (let i = 1; i < 6; i++) {
+    setTimeout(() => {
+      addListElement(i);
+    }, i * 1500);
+  }
+  setTimeout(() => {
+    addFeedbackResult('POSITIVE');
+  }, 9000);
+
+  for (let i = 0; i < 6; i++) {
+    const values = [
+      'TOXICITY',
+      'IDENTITY ATTACK',
+      'INSULT',
+      'PROFANITY',
+      'THREAT',
+      'FLIRTATION',
+    ];
+    addDonutChart(values[i], Math.floor(Math.random() * 61) + 30);
+  }
+};
+
+/**
+ * Adds an bullet point to the comment list
+ *
+ * @param {string} str the txt of the bullet point
+ */
+function addListElement(str) {
+  const el = document.createElement('li');
+  el.innerHTML = str;
+  document.getElementById('list').appendChild(el);
+}
+
+/**
+ * Adds the feedback result element to the DOM tree
+ *
+ * @param {string} result Positive, Negative, or Neutral
+ */
+function addFeedbackResult(result) {
+  const el = document.createElement('div');
+  el.classList = 'center fade';
+  const text = document.createElement('h1');
+  text.setAttribute('id', 'overall');
+  text.innerHTML = 'Overall Response\n: ' + result;
+  el.appendChild(text);
+
+  const list = document.getElementById('list');
+  list.appendChild(el);
+}
+
+/**
+ * Adds a donut chart
+ *
+ * @param {string} str the header string of the circle
+ * @param {string} percent the amount the circle will be filled
+ */
+function addDonutChart(str, percent) {
+  const div = document.createElement('div');
+
+  div.setAttribute('id', 'a-chart');
+  div.className = 'item donut';
+  div.style.setProperty('--percent', percent);
+
+  const header = document.createElement('h2');
+  header.innerText = str;
+
+  div.appendChild(header);
+
+  const svg = createSVGElement('svg');
+
+  svg.setAttribute('xmlns', 'http://www.w3.org/2000/svg');
+
+  const g = buildCircle();
+  svg.appendChild(g);
+
+  div.appendChild(svg);
+
+  const currentDiv = document.getElementById('charts');
+  currentDiv.appendChild(div);
```

Comment:
too many spaces. reorganize for readability. Each block should be discrete. (readjust as needed)
```
  const header = document.createElement('h2');
  header.innerText = str;
  div.appendChild(header);

  const svg = createSVGElement('svg');
  svg.setAttribute('xmlns', 'http://www.w3.org/2000/svg');

  const g = buildCircle();
  svg.appendChild(g);

  div.appendChild(svg);

  const currentDiv = document.getElementById('charts');
  currentDiv.appendChild(div);
```